ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/TopKKudafBaseTest.java

```diff
@@ -0,0 +1,16 @@
+package io.confluent.ksql.function.udaf.topk;
+
+import io.confluent.ksql.function.KsqlAggregateFunction;
+import org.apache.kafka.connect.data.Schema;
+
+import java.util.List;
+
+public abstract class TopKKudafBaseTest {
```

Comment:
I don't see the point of this abstract class. It isn't adding anything

Reply:
The current test is quite limited tests only small size arrays. The idea is to introduce array random generators arbitrary length and test TOPK and TOPKDISTINCT on larger arrays.

Reply:
I'd just fold this into the other tests.  I don't think it is adding any value

Reply:
I've removed this class from the PR. @dguy